### PR TITLE
Removed: ``PageContainer`` does not wrap its child into a ``<div>``

### PR DESCRIPTION
**Pull request by MoOx**
Closes #690 

poke @DavidWells

### CODE REVIEW

**Comment by MoOx** [comment]
I think I have a better idea: we should remove this div. It's useless. We should not render multiple components in PageContainer anyway.
Does this looks good to you?

**Comment by thangngoc89** [comment]
LGTM

**Comment by MoOx** [comment]
Updated. What do you think?

File: src/PageContainer/component.js [hunk 1]
```diff
@@ -233,29 +233,27 @@ class PageContainer extends Component<DefaultProps, Props, void> {
...
+    else {
+      if (page.loading && PageLoading) {
+        return <PageLoading />
+      }
+      else if (Layout) {
+        return <Layout ref={ this.saveContentRef } { ...page } />
+      }
+    }
+
+    return null
```
Comment (thangngoc89) [inline src/PageContainer/component.js]: Is this valid ? Never return null from React render

**Comment by thangngoc89** [comment]
It's much cleaner now. But I saw these changes in the [feat.prepare-theme-base](https://github.com/MoOx/phenomic/tree/feat.prepare-theme-base) too. Rebased?

File: src/PageContainer/component.js (hunk 1 above)
Comment (MoOx) [inline src/PageContainer/component.js]: Yeah it's totally valid, React render null as... nothing! Handy for component that can skip their render themselves.
But in our case in theory, this should not happen :)

**Comment by MoOx** [comment]
Last commit of this branch was a totally mess, it was to show this to a friend. I am editing this branch ;)

File: src/PageContainer/component.js (hunk 1 above)
Comment (thangngoc89) [inline src/PageContainer/component.js]: I learned new thing today. Thanks :100:

Comment (MoOx) [inline src/PageContainer/component.js]: This can be convenient for stuff like a `<LoadingIndicator loading={ boolean } />` :D

**Pull request merged**